includes/wf_crm_admin_form.inc

```diff
@@ -1031,6 +1030,37 @@ class wf_crm_admin_form {
         }
       }
     }
+    // KG - start
+    // LineItem
+    $c = 1;
```

Comment:
I don't think we need this variable. There's only one contribution. Ever.

Reply:
I kept it thinking we may want to at some point have more than 1 contribution - but yeah - will simplify.